src/simulator_core/adapter/transforms/esdl_asset_mapper.py

```diff
@@ -31,6 +31,7 @@ class EsdlAssetMapper:
     conversion_dict = {
         esdl.Producer: ProductionCluster,
         esdl.GenericProducer: ProductionCluster,
+        esdl.GeothermalSource: ProductionCluster,
```

Comment:
This addition should only be added in the respective PR.

Reply:
I added this due to my example is using geothermal source, i will remove it then